libmscore/chordrest.cpp

```diff
@@ -1144,9 +1154,9 @@ bool ChordRest::isGrace() const
 bool ChordRest::isGraceBefore() const
       {
       return isChord()
-         && (toChord(this)->noteType() & (
-           NoteType::ACCIACCATURA | NoteType::APPOGGIATURA | NoteType::GRACE4 | NoteType::GRACE16 | NoteType::GRACE32
-           ));
+                  && (toChord(this)->noteType() & (
+                            NoteType::ACCIACCATURA | NoteType::APPOGGIATURA | NoteType::GRACE4 | NoteType::GRACE16 | NoteType::GRACE32
+                            ));
```

Comment:
The indent before looked OK to me...

Reply:
Explain that the qtCreator Ctrl+I auto indenting.... (USING the preferred musescore settings file)